orttraining/orttraining/test/training_ops/cuda/optimizer/adamw_test.cc

```diff
@@ -0,0 +1,928 @@
+                  {3.51157e-05f, 4.19913e-05f, 3.4092e-06f, 6.4173e-06f, 0.0001198826f, 5.33027e-05f},
+                  {5.41122e-05f, 4.41312e-05f, 3.4058e-06f, 6.4109e-06f, 0.0001197627f, 5.32494e-05f},
+                  {8.43036e-05f, 0.0003451189f, 3.4024e-06f, 6.4045e-06f, 0.0001196429f, 5.31962e-05f},
+                  {0.0004303346f, 0.0009178877f, 3.36719e-05f, 5.77829e-05f, 0.0001222358f, 5.77471e-05f},
+                  {0.0004566041f, 0.0009199848f, 3.36382e-05f, 5.77251e-05f, 0.0001270772f, 5.9526e-05f},
+                  {0.0004583648f, 0.000954971f, 0.0002876955f, 9.16751e-05f, 0.0001272302f, 8.95574e-05f},
+                  {0.0004644046f, 0.0009589141f, 0.0002874078f, 9.15834e-05f, 0.0001271029f, 8.94678e-05f},
+                  {0.000464441f, 0.0009633462f, 0.0002871204f, 9.14918e-05f, 0.0001269758f, 8.93783e-05f},
+              },
+          },
+          {
+              "fc1.bias",
+              {
+                  {0.0f, 0.0f, 0.0f},
+                  {9.54826e-05f, 0.0f, 0.0001327461f},
+                  {0.0001216516f, 6.409e-07f, 0.0002738411f},
+                  {0.0001393831f, 6.6671e-06f, 0.0002738729f},
+                  {0.0002350543f, 6.6604e-06f, 0.000273599f},
+                  {0.0003172084f, 6.6538e-06f, 0.0002733254f},
+                  {0.0003225325f, 3.34178e-05f, 0.0002754508f},
+                  {0.0004695006f, 3.33844e-05f, 0.0003100878f},
+                  {0.0004999468f, 0.0002439171f, 0.0003486728f},
+                  {0.0005020121f, 0.0002436732f, 0.0003483242f},
+                  {0.0005148426f, 0.0002434295f, 0.0003479759f},
+              },
+          },
+          {
+              "fc2.weight",
+              {
+                  {0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f},
+                  {0.0001012934f, 0.0f, 1.04611e-05f, 0.0006197428f, 0.0f, 4.15992e-05f},
+                  {0.0001426376f, 1.687e-07f, 1.05037e-05f, 0.0011384432f, 4.63e-07f, 4.23176e-05f},
+                  {0.0001608251f, 1.39469e-05f, 1.10024e-05f, 0.0011418851f, 3.7832e-06f, 4.28334e-05f},
+                  {0.0002771204f, 1.39329e-05f, 1.09914e-05f, 0.0012316541f, 3.7794e-06f, 4.27905e-05f},
+                  {0.0003728576f, 1.3919e-05f, 1.09804e-05f, 0.0033274253f, 3.7757e-06f, 4.27477e-05f},
+                  {0.0004274074f, 0.0003023939f, 1.86132e-05f, 0.0033410995f, 7.58887e-05f, 4.46158e-05f},
+                  {0.0005188234f, 0.0003020915f, 1.87601e-05f, 0.003762914f, 7.58128e-05f, 4.46979e-05f},
+                  {0.0005690802f, 0.0003018424f, 2.76857e-05f, 0.0051242807f, 9.95882e-05f, 8.2914e-05f},
+                  {0.0005691052f, 0.0003015406f, 2.7658e-05f, 0.0054629003f, 9.94886e-05f, 8.28311e-05f},
+                  {0.0005827378f, 0.000301239f, 2.76304e-05f, 0.005508867f, 9.93891e-05f, 8.27482e-05f},
+              },
+          },
+          {
+              "fc2.bias",
+              {
+                  {0.0f, 0.0f},
+                  {0.0005453224f, 0.0029798434f},
```

Comment:
I feel it will be a lot cleaner if we read these from some file

Reply:
This is something I wanted to change, let me check whether it is easier to do it or not.

Reply:
Fixed.